pkg/systemd/overview-cards/cryptoPolicies.jsx

```diff
@@ -223,7 +223,7 @@ export const CryptoPolicyStatus = () => {
     if (isInconsistentPolicy(currentCryptoPolicy, fipsEnabled)) {
         return (
             <li className="system-health-crypto-policies">
-                <Flex spacer={{ default: 'spaceItemsSm' }} flexWrap={{ default: 'nowrap' }}>
```

Comment:
This also appears in two more places:

pkg/systemd/overview-cards/lastLogin.jsx:            <Flex spacer={{ default: 'spaceItemsSm' }} flexWrap={{ default: 'nowrap' }}>
pkg/systemd/overview-cards/shutdownStatus.jsx:            <Flex spacer={{ default: 'spaceItemsSm' }} flexWrap={{ default: 'nowrap' }}>

Plus, are these valid?

pkg/users/account-create-dialog.js:                    <FlexItem spacer={{ default: 'spacerNone' }}>
pkg/users/account-create-dialog.js:                    <FlexItem spacer={{ default: 'spacerNone' }}>
pkg/users/account-details.js:                                        <FlexItem spacer={{ default: 'spacerNone' }}>